docs/dataservice-read-catalog-example.md

```diff
@@ -180,9 +179,9 @@ The `OlpClientSettings` class pulls together all the different settings for cust
 
 Configuration of the `authentication_settings` is sufficient to start using the `CatalogClient`.
 
-### Retrieve catalog metadata
+### Retrieve catalog configuration
 
-To retrieve catalog metadata, create a `CatalogRequest`. The `CatalogRequest` class allows to set properties of the catalog request, including:
+To retrieve catalog configuration, create a `CatalogRequest`. The `CatalogRequest` class allows to set properties of the catalog request, including:
 
 * `WithBillingTag`: Sets the billing tag used for this request.
```

Comment:
We can combine it with the previous sentence (see my previous comment). No need in the bulleted list here since we have only one list item.